### PR TITLE
[Dashboards-desktop] Update electron version

### PR DESCRIPTION
**Pull request by Flyingliuhub**
Signed-off-by: Tao Liu <liutaoaz@amazon.com>

### Description
Update electron version to resolve this [issue](https://nvd.nist.gov/vuln/detail/CVE-2022-36077)

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/johnnyon-amzn/dashboards-desktop-poc#readme",
   "devDependencies": {
-    "electron": "^18.1.0"
+    "electron": "18.3.7"
```
Comment (zhongnansu) [inline package.json:20]: why not use `^18.3.7`

**Review by seraphjiang — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Flyingliuhub) [reply package.json:20]: > why not use `^18.3.7`

https://nvd.nist.gov/vuln/detail/CVE-2022-36077, "This issue has been patched in versions: 21.0.0-beta.1, 20.0.1, 19.0.11, and 18.3.7."

[the remark this answers is not in the document]
Comment (Flyingliuhub) [reply package.json:20]: https://nvd.nist.gov/vuln/detail/CVE-2022-36077, "This issue has been patched in versions: 21.0.0-beta.1, 20.0.1, 19.0.11, and 18.3.7."

**Comment by seanneumann** [comment]
Thanks @Flyingliuhub !

**Pull request merged**